plugin/adsk/plugin/exportTranslator.cpp

```diff
@@ -93,6 +93,7 @@ UsdMayaExportTranslator::writer(const MFileObject &file,
                 theOption[1].split(',', filteredTypes);
             }
             else if (argName == "frameSample") {
```

Comment:
Does this mean we can have the same argument name twice?
Additionally, do we have additional copy of this code somewhere? (potentially Pixar plugin?)